cdap-docs/developer-manual/source/cloud-runtimes/profiles/assigning-profiles.rst

```diff
@@ -8,5 +8,75 @@
+assign a profile to a batch pipeline that will be used for runs that are manually started,
+and you can assign a profile to a pipeline schedule.
 
+The profile that is actually used at runtime is determined by walking the CDAP hierarchy
+from most general to most specific.
+
+instance > namespace > schedule/manual run
+
+If a profile is set on the schedule that triggered a run, that profile will be used.
+If no profile is set, the default profile for the namespace will be used. If no default
+is set on the namespace, the default for CDAP will be used. If no CDAP default is set,
+the native profile will be used.
+
+Similarly, if a pipeline is manually run and there is a profile assigned to that pipeline,
+that profile will be used.
+If no profile is set, the default profile for the namespace will be used. If no default
+is set on the namespace, the default for CDAP will be used. If no CDAP default is set,
+the native profile will be used.
+
+Note: If you have deployed your own custom applications and are using workflows, you
+can also use profiles to run your workflow in a cloud environment. Batch pipelines are
+just workflows, but have special UI support to make it easier to use profiles with them.
+ 
+Assigning Default Profiles
+--------------------------
+Default profiles can be assigned to a namespace and to the CDAP instance.
+This can be done through the UI by navigating to the Administration page and clicking
+on the star next to the profile.
+
+Alternatively, the :ref:`Preferences RESTful API <http-restful-api-preferences>` can be used
+to set default profiles. To set the default profile for all of CDAP, set a preference on the
+CDAP instance with key `profile` and value `system:<profile-name>`. To set the default profile for a
+namespace, set a preference on the desired namespace with key `profile` and value `<scope>:<profile-name>` 
+
+Assigning a Profile for Manual Runs
+-----------------------------------
+You can assign a profile to use for manual pipeline runs by navigating to the pipeline detail page.
+There, you can click on the configure button, Advanced Options, then Compute config. From that section,
+you can select a profile and save. From that point on, any time the pipeline is manually run, that
+profile will be used.
+
+Alternatively, the :ref:`Preferences RESTful API <http-restful-api-preferences>` can be used
+to set the profile for manual runs. To do so, set a preference on the `DataPipelineWorkflow` entity
+with key `profile` and value `<scope>:<profile-name>`.
+
+Assigning a Profile to a Schedule
+---------------------------------
+Any time you create a schedule for a pipeline, you can assign a profile to it. Whenever a pipeline
+run is triggered by that schedule, it will use that profile for the run. This is true for time
+schedules as well as schedules that are triggered by other pipelines.
+
+Alternatively, the :ref:`Schedules RESTful API <http-restful-api-lifecycle-schedule-add>` can be used
```

Comment:
Alternatively, you can use the :ref:`Schedules RESTful API <http-restful-api-lifecycle-schedule-add>` to assign a profile to a schedule.